products/batch_exports/backend/tests/temporal/destinations/snowflake/test_activity_e2e.py

```diff
@@ -657,12 +661,20 @@ async def test_insert_into_snowflake_activity_handles_uppercased_columns(
         data_interval_end=data_interval_end,
         table_name=table_name,
         batch_export_model=model,
-        sort_key="uuid",
+        sort_key=sort_key,
     )
 
+    uppercase_columns = ["team_id"]
+    if model.name == "events":
+        uppercase_columns.append("event")
+    elif model.name == "persons":
+        uppercase_columns.append("person_version")
+        uppercase_columns.append("person_id")
     snowflake_cursor.execute(f'TRUNCATE TABLE "{table_name}"')
-    snowflake_cursor.execute(f'ALTER TABLE "{table_name}" RENAME COLUMN "event" TO "EVENT"')
-    snowflake_cursor.execute(f'ALTER TABLE "{table_name}" RENAME COLUMN "timestamp" TO "TIMESTAMP"')
+    for column in uppercase_columns:
+        snowflake_cursor.execute(f'ALTER TABLE "{table_name}" RENAME COLUMN "{column}" TO "{column.upper()}"')
+    # person_id is now in uppercase
+    sort_key = "uuid" if model.name == "events" else "PERSON_ID"
```

Comment:
Does this work? We use the same sort key for both records read from clickhouse and those fetched from snowflake. But the ones in Clickhouse are not uppercase. Hence why when I wrote this test I didn't uppercase uuid as that's the sort key for events.